showcase/im_037676_rgb.xml

```diff
@@ -0,0 +1,2 @@
x>479</xmax><ymax>1292</ymax></bndbox></object><object><name>parph</name><bndbox><xmin>1762</xmin><ymin>1198</ymin><xmax>1779</xmax><ymax>1219</ymax></bndbox></object><object><name>parph</name><bndbox><xmin>1283</xmin><ymin>1195</ymin><xmax>1310</xmax><ymax>1213</ymax></bndbox></object><object><name>parph</name><bndbox><xmin>1770</xmin><ymin>1142</ymin><xmax>1796</xmax><ymax>1167</ymax></bndbox></object><object><name>parph</name><bndbox><xmin>1972</xmin><ymin>1140</ymin><xmax>1985</xmax><ymax>1167</ymax></bndbox></object><object><name>parph</name><bndbox><xmin>1588</xmin><ymin>1060</ymin><xmax>1622</xmax><ymax>1085</ymax></bndbox></object><object><name>parph</name><bndbox><xmin>1974</xmin><ymin>900</ymin><xmax>1995</xmax><ymax>920</ymax></bndbox></object><object><name>parph</name><bndbox><xmin>68</xmin><ymin>897</ymin><xmax>94</xmax><ymax>924</ymax></bndbox></object><object><name>parph</name><bndbox><xmin>758</xmin><ymin>868</ymin><xmax>789</xmax><ymax>905</ymax></bndbox></object><object><name>parph</name><bndbox><xmin>428</xmin><ymin>778</ymin><xmax>453</xmax><ymax>814</ymax></bndbox></object><object><name>parph</name><bndbox><xmin>1844</xmin><ymin>569</ymin><xmax>1872</xmax><ymax>594</ymax></bndbox></object><object><name>parph</name><bndbox><xmin>1681</xmin><ymin>507</ymin><xmax>1717</xmax><ymax>545</ymax></bndbox></object><object><name>parph</name><bndbox><xmin>1887</xmin><ymin>491</ymin><xmax>1914</xmax><ymax>520</ymax></bndbox></object><object><name>parph</name><bndbox><xmin>41</xmin><ymin>450</ymin><xmax>64</xmax><ymax>470</ymax></bndbox></object><object><name>parph</name><bndbox><xmin>1932</xmin><ymin>392</ymin><xmax>1979</xmax><ymax>437</ymax></bndbox></object><object><name>parph</name><bndbox><xmin>1112</xmin><ymin>384</ymin><xmax>1160</xmax><ymax>429</ymax></bndbox></object><object><name>parph</name><bndbox><xmin>1404</xmin><ymin>364</ymin><xmax>1446</xmax><ymax>398</ymax></bndbox></object><object><name>parph</name><bndbox><xmin>1933</xmin><ymin>353</ymin><xmax>1958</xmax><ymax>384</ymax></bndbox></object><object><name>parph</name><bndbox><xmin>1457</xmin><ymin>325</ymin><xmax>1487</xmax><ymax>360</ymax></bndbox></object><object><name>parph</name><bndbox><xmin>1037</xmin><ymin>287</ymin><xmax>1071</xmax><ymax>309</ymax></bndbox></object><object><name>parph</name><bndbox><xmin>641</xmin><ymin>199</ymin><xmax>655</xmax><ymax>227</ymax></bndbox></object><object><name>parph</name><bndbox><xmin>1274</xmin><ymin>185</ymin><xmax>1291</xmax><ymax>206</ymax></bndbox></object><object><name>parph</name><bndbox><xmin>50</xmin><ymin>178</ymin><xmax>98</xmax><ymax>226</ymax></bndbox></object><object><name>parph</name><bndbox><xmin>1867</xmin><ymin>161</ymin><xmax>1905</xmax><ymax>186</ymax></bndbox></object><object><name>parph</name><bndbox><xmin>1083</xmin><ymin>129</ymin><xmax>1118</xmax><ymax>162</ymax></bndbox></object><object><name>parph</name><bndbox><xmin>328</xmin><ymin>107</ymin><xmax>344</xmax><ymax>143</ymax></bndbox></object></annotation>
```

Comment:
Je to slité do jednoho řádku a ještě nekorektně ukončené.

Reply:
Chceme ty xml v gitu? Chápu, chtěl jsem tu vzorové data, ale i tak bych tu nechal jen obrázky. Možná, kdyby to byla součást unittestu.

Reply:
Ty xml dej pryč, nebo alespoň přegeneruj.